nerfstudio/data/datamanagers/base_datamanager.py

```diff
@@ -117,6 +117,8 @@ class DataManagerConfig(InstantiateConfig):
     """Specifies the camera pose optimizer used during training. Helpful if poses are noisy."""
     masks_on_gpu: Optional[bool] = None
     """Process masks on GPU for speed at the expense of memory, if True."""
+    images_on_gpu: Optional[bool] = None
```

Comment:
Doesn't need to be `Optional`. Use `images_on_gpu: bool = False`.
The same should be true for `masks_on_gpu`